INSTALL.rst

```diff
@@ -74,4 +74,51 @@ Now you can visit http://localhost:5000/ :)
 Docker
 ------
 
-Soon ...
+First clone the repository, if you haven't done it already, build all docker
+images and boot them up using Docker Compose:
+
+.. code-block:: console
+
+    $ git clone https://github.com/CERNDocumentServer/cds.git
+    $ git checkout master
+    $ docker-compose -f docker-compose.yml build
```

Comment:
You don't need to specify the filename (`docker-compose.yml` is the default filename) :)